source/extensions/filters/listener/proxy_protocol/proxy_protocol.h

```diff
@@ -43,6 +43,17 @@ using ConfigSharedPtr = std::shared_ptr<Config>;
 
 enum ProxyProtocolVersion { Unknown = -1, InProgress = -2, V1 = 1, V2 = 2 };
 
+enum class ProxyState{
+    //failed to read proxy protocol
+    PrtclFail,
+
+    //Unsupported command or address family or transport
+    UnsupportedCd,
+
+    //the method's job is over
```

Comment:
Please fix format of comments, add a space after '/' and a period please. I think fix format script will do this.

Reply:
I ran the lines in the link you've provided; however, they did not work; so I've manually fixed them.